tools/wpt/browser.py

```diff
@@ -431,8 +431,40 @@ class Servo(Browser):
     product = "servo"
     requirements = "requirements_servo.txt"
 
-    def install(self, dest=None):
-        raise NotImplementedError
+    def platform_string(self):
+        platform = {
+            "Linux": "linux",
+            "Windows": "win",
+            "Darwin": "mac"
+        }.get(uname[0])
+
+        if platform is None:
+            raise ValueError("Unable to construct a valid Servo package for current platform")
+
+        if platform == "linux":
+            bits = "-%s" % uname[4]
+            extension = ".tar.gz"
+        elif platform == "win":
+            bits = "64" if uname[4] == "x86_64" else "32"
+            extension = ".msi"
+        else:
+            bits = ""
+            extension = ".dmg"
+
+        return "%s%s" % (platform, extension)
+
+    def install_browser_engine(self, dest=None):
```

Comment:
You renamed the `install` method to `install_browser_engine`, so the error message is letting you know that it expects the `install` method.

Reply:
Thanks I have corrected this. This is what it shows now.

![testm1](https://user-images.githubusercontent.com/19872082/37157353-a613db0e-230e-11e8-9fa4-bd3db25a3549.png)

Reply:
That looks like you restored the original `install` method to its original implementation, which just raises a NotImplementedError exception.

Reply:
Also, rather than attaching screenshots, I recommend copying the text out of your console and pasting it between blocks of ``` characters like this:
```
jdm@godot$ ./wpt run
```

Reply:
@jdm I. Isn't this the implementation of install method or I have done something wrong here? Why is it giving NotImplementedError after renaming it to `install` ?

Reply:
@kriti21 It's hard to say without looking at the actual code you're running. Please push your changes if you'd like me to comment on them :)